lib/OpenCloud/LoadBalancer/Service.php

```diff
@@ -51,11 +52,20 @@ public function loadBalancer($id = null)
      */
     public function loadBalancerList($detail = true, array $filter = array())
     {
+        $options = $this->makeResourceIteratorOptions($this->resolveResourceClass('LoadBalancer'));
+
+        if (isset($filter['limit'])) {
+            $options['limit.page'] = $filter['limit'];
+            unset($filter['limit']);
+        }
+
         $url = $this->getUrl();
         $url->addPath(Resource\LoadBalancer::resourceName());
         $url->setQuery($filter);
 
-        return $this->resourceList('LoadBalancer', $url);
+        $options += array('baseUrl' => $url, 'key.marker' => 'id');
```

Comment:
Yep but in this case the options are known to only contain a few keys (see [here](https://github.com/rackspace/php-opencloud/blob/working/lib/OpenCloud/Common/Base.php#L406-L416)) so I don't think there's a risk of collisions but I'll change anyway